roles/manage_patches/tasks/validate_efm_cluster.yml

```diff
@@ -0,0 +1,75 @@
+---
+- name: Get the primary based on inventory details
+  set_fact:
+    efm_inventory_primary: "{{ node.inventory_hostname }}"
+  loop: "{{ efm_cluster_nodes }}"
+  loop_control:
+    loop_var: node
+  when:
+  - use_hostname|bool
+  - node.node_type == 'primary'
+  run_once: true
+  no_log: "{{ disable_logging }}"
+
+- name: Get the primary based on inventory details
+  set_fact:
+    efm_inventory_primary: "{{ node.private_ip }}"
+  loop: "{{ efm_cluster_nodes }}"
+  loop_control:
+    loop_var: node
+  when:
+  - not use_hostname|bool
+  - node.node_type == 'primary'
+  run_once: true
+  no_log: "{{ disable_logging }}"
+
+- name: Execute efm cluster status
+  ansible.builtin.shell: >
+    set -o pipefaili;
+    /usr/edb/efm-4.4/bin/efm cluster-status-json main
```

Comment:
Use the variable `efm_cluster_name` instead of `main`

Reply:
fixed